gen/indykite/identity/v1beta2/attributes.ext.go

```diff
@@ -0,0 +1,227 @@
+			return anyVal.AnyValue.UnmarshalTo(msg)
+		} else if _, ok = val.ObjectValue.GetValue().(*objects.Value_NullValue); ok {
+			return nil
+		}
+	}
+	return errors.New("invalid AnyPB value")
+}
+
+func (x *Property) GetMapValue() (val map[string]interface{}, err error) {
+	if val, ok := x.Value.(*Property_ObjectValue); ok {
+		if mapVal, ok := val.ObjectValue.GetValue().(*objects.Value_MapValue); ok {
+			return objects.ToMap(mapVal.MapValue.Fields)
+		} else if _, ok = val.ObjectValue.GetValue().(*objects.Value_NullValue); ok {
+			return nil, nil
+		}
+	}
+	return nil, errors.New("invalid map value")
+}
+
+// validateValue checks if an operation value is properly set.
+//
+// It checks if the property values is not nil.
+func (PropertyBatchOperations) validateValue(value isProperty_Value, enableNil bool) error {
+	switch propertyValue := value.(type) {
+	case nil:
+		if !enableNil {
+			return errors.New("value cannot be nil")
+		}
+	case *Property_ObjectValue:
+		if propertyValue.ObjectValue == nil || propertyValue.ObjectValue.Value == nil {
+			return errors.New("object value must be specified")
+		}
+	case *Property_ReferenceValue:
+		if len(propertyValue.ReferenceValue) == 0 {
+			return errors.New("reference value must be specified")
+		}
+	default:
+		return errors.New("invalid value type")
+	}
+	return nil
+}
+
+func (PropertyBatchOperations) validateMeta(meta *PropertyMetadata, isTrusted bool) error {
+	if meta == nil {
+		return nil
+	}
+
+	if meta.AssuranceLevel > AssuranceLevel_ASSURANCE_LEVEL_LOW && meta.Verifier == "" {
+		return errors.New("verifier is required when assurance level is Substantial or more")
+	}
+	if meta.Verifier != "" && meta.AssuranceLevel < AssuranceLevel_ASSURANCE_LEVEL_SUBSTANTIAL {
+		return errors.New("assurance level must be Substantial or more when verifier is set")
+	}
+	if meta.Verifier == "" && meta.VerificationTime != nil {
+		return errors.New("verification time must be set together with verifier")
+	}
+
+	// If is non-trusted, we don't care about issuer as it will be ignored
+	if isTrusted && meta.Issuer != "" {
+		switch {
+		case len(meta.Issuer) == 26 &&
+			(meta.Issuer[:3] == string(PropertyApplicationPrefix) || meta.Issuer[:3] == string(PropertyAppSpacePrefix)):
+			if uuid.UUID(meta.Issuer[4:]).Variant() != uuid.RFC4122 {
+				return errors.New("issuer must be RFC4122 variant UUID")
+			}
+		case len(meta.Issuer) >= 7 && meta.Issuer[:3] == string(PropertyRawPrefix):
+			// This is valid
+		default:
+			return errors.New("issuer must starts with 'app:' or 'asp:' followed by trimmed base64URL UUID," +
+				" or starts with 'raw:' followed by name with minimum length of 3 characters")
+		}
+	}
+
+	return nil
+}
+
+// Validate checks the PropertyBatchOperations values. All public calls are considered non-trusted.
+//
+// It checks if the operations represent a valid collection of property updates.
+func (x PropertyBatchOperations) Validate(isTrusted bool) error {
```

Comment:
Remove the whole `isTrusted` parametr and update code as this is always `false`. isTrusted=true only in BE calls.